src/main/java/rs/teslaris/core/controller/OrganisationUnitRelationController.java

```diff
@@ -0,0 +1,90 @@
+package rs.teslaris.core.controller;
+
+import java.util.List;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import rs.teslaris.core.dto.document.DocumentFileDTO;
+import rs.teslaris.core.dto.institution.OrganisationUnitsRelationDTO;
+import rs.teslaris.core.dto.institution.OrganisationUnitsRelationResponseDTO;
+import rs.teslaris.core.service.OrganisationUnitService;
+
+@RestController
+@RequestMapping("/api/relation")
```

Comment:
again here we could have an issue with paths. I could imagine that we have later relations between documents, if all is in the shape {app}/api/relation/23432/766 could we have some issues?

Reply:
Changed to "/api/organisation-unit-relations" for now.

Maybe another option is to merge the controller into OrganisationUnitController so the path would be "/api/organisation-units/relations"